lib/plugins/exchange/bitstamp.js

```diff
@@ -18,6 +18,9 @@ const loadConfig = (account) => {
   return { ...mapped, timeout: 3000 }
 }
 
-const isConfigValid = ({ key, clientId, secret }) => key && secret && clientId
+const isConfigValid = options => {
+  const requiredOptions = _.pick(['key', 'secret', 'clientId'], options)
+  return _.isEqual(options, requiredOptions)

```

Comment:
I'm not following this logic here, just testing if the original options is equal to the picked option but they all can still be null.